src-electron/sdk/matter.js

```diff
@@ -23,26 +23,11 @@ const dbEnum = require('../../src-shared/db-enum.js')
 const fs = require('fs')
 const fsp = fs.promises
 
-function parseJson(json) {
-  try {
-    return JSON.parse(json)
-  } catch (err) {
-    return undefined
-  }
-}
-
 /**
- * This file implements upgrade rules which are used to upgrade .zap files and xml files
- * to be in sync with the spec
+ * This file implements SDK rules which are used to upgrade .zap files and xml files
+ * to be in sync with the Matter SDK and spec
```

Comment:
Does this comment need to be removed along with the function definition?

Reply:
no it does the same thing just in a slightly different way